gradle.properties

```diff
@@ -5,4 +5,4 @@ jacocoVersion=0.7.1.201405082137
 
 # test compile dependencies
 junitVersion=4.11
-jacksonVersion=2.5.1
+jacksonVersion=2.4.4
```

Comment:
Note that this was my suggestion. It follows https://github.com/palantir/http-remoting/pull/57 (spark needing jackson 2.4)